lib/dmc/fsr_configuration.rb

```diff
@@ -1,31 +1,11 @@
 # frozen_string_literal: true
 
-module DMC
-  class FSRConfiguration < Common::Client::Configuration::REST
-    def self.base_request_headers
-      super.merge(
-        'client_id' => Settings.dmc.client_id,
-        'client_secret' => Settings.dmc.client_secret
-      )
-    end
+require 'dmc/base_configuration'
 
+module DMC
+  class FSRConfiguration < DMC::BaseConfiguration
```

Comment:
Would prefer to see `FinancialStatusReportConfiguration` to reduce confusion/inflection issues